src/quadrature/Ordinate_Set.hh

```diff
@@ -59,29 +59,31 @@ public:
   // ACCESSORS
 
   //! Return the ordinates.
-  vector<Ordinate> const &ordinates() const { return ordinates_; }
+  vector<Ordinate> const &ordinates() const noexcept { return ordinates_; }
```

Comment:
Marking this function `noexcept` seems like a really bad idea to me.   `ordinates_` may not even be allocated and yet you are returning the address to it.  I guess I need to be convinced that is is a good idea.  Am I missing a larger design or performance aspect related to this change?

From what I am reading about this pattern for _getters_ (this might even be an _antipattern_ but I'm not certain), they shouldn't be marked `noexcept` except in [cases where you can demonstrate an extreme performance benefit](https://stackoverflow.com/questions/21330807/should-i-use-noexcept-for-getters-always).

Also, I am [reading](https://stackoverflow.com/questions/10787766/when-should-i-really-use-noexcept) that 

> Most compilers follow a no-penalty-if-you-don't-throw exception handling method, so I doubt it would change much (or anything) on the machine code level of your code, although perhaps reduce the binary size by removing the handling code. 

So again, I'm not convinced that any of these changes will make a significant performance improvement and they add complexity where it isn't necessarily merited.